now/hub/most_simple_indexer/most_simple_indexer.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+
+from docarray import DocumentArray
+from jina import Executor, requests
+
+
+class MostSimpleIndexer(Executor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.index = DocumentArray()
+
+    @requests
+    def index(self, docs: Optional[DocumentArray], **kwargs):
```

Comment:
`docs` are no longer optional
```suggestion
    def index(self, docs: DocumentArray, **kwargs):
```